packages/ocamlgraph_gtk/ocamlgraph_gtk.2.0.0/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+synopsis: "Displaying graphs using OCamlGraph and GTK"
+description: "Displaying graphs using OCamlGraph and GTK"
+maintainer: ["filliatr@lri.fr"]
+authors: ["Sylvain Conchon" "Jean-Christophe Filliâtre" "Julien Signoles"]
+license: "LGPL-2.1-only"
+tags: [
+  "graph"
+  "library"
+  "algorithms"
+  "directed graph"
+  "vertice"
+  "edge"
+  "persistent"
+  "imperative"
+]
+homepage: "https://github.com/backtracking/ocamlgraph/"
+bug-reports: "https://github.com/backtracking/ocamlgraph/issues/new"
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "stdlib-shims"
+  "lablgtk"
+  "conf-gnomecanvas"
+  "ocamlgraph"
+  "dune" {>= "2.0"}
```

Comment:
```suggestion
  "dune" {>= "2.0" & !with-test | >= "2.8"}
```

Reply:
done.

Many thanks @dra27 for all these suggestions. You have to pardon me for my inexperience with `dune` and `opam` packaging; I'm relatively new to these things.

Reply:
No problem at all! The graphics/Dune thing is an unusual corner-case anyway. It's lovely to have ocamlgraph ported to build with Dune (having maintained patches for it in opam for quite a while!)